src/Modules/UngradedModules/FreeResolutions.jl

```diff
@@ -240,11 +240,43 @@ Return a free resolution of `M`.
 
 If `length != 0`, the free resolution is only computed up to the `length`-th free module.
 Current options for `algorithm` are `:fres`, `:nres`, and `:mres` for modules over
-polynomial rings and `:sres` for modules over quotients of polynomial rings.
+polynomial rings and `:sres` for modules over quotients of polynomial rings (see below).
 
 !!! note
     The function first computes a presentation of `M`. It then successively computes
-    higher syzygy modules.
+    higher syzygy modules. In the illustrating example below, the free resolution is
+    first computed up to length 1:
+    ```@julia
```

Comment:
could you replace these code snippets by `jldoctest`s to make sure they stay correct over time?